lib/matplotlib/projections/geo.py

```diff
@@ -146,8 +156,15 @@ def set_yscale(self, *args, **kwargs):
         if args[0] != 'linear':
             raise NotImplementedError
 
+    # Prevent the user from applying scales to one or both of the
+    # axes.  In this particular case, scaling the axes wouldn't make
```

Comment:
"in this particular case, scaling the axes wouldn't make sense" 'made me do a doable take and also wondering if it's needed here since the goal is just the first line. Basically, I think trying to explain it might introduce more confusion since your explanation is basically it's self explanatory.

Reply:
fixed